src/test/objectstore/CMakeLists.txt

```diff
@@ -124,7 +124,6 @@ endif(WITH_BLUESTORE)
 add_executable(unittest_transaction
   test_transaction.cc)
 add_ceph_unittest(unittest_transaction)
-target_link_libraries(unittest_transaction os ceph-common ${XIO_LIBRARY})
```

Comment:
we still need to link against `os` and `ceph-common`.

Reply:
I'm in favor of dropping xiomessenger, and I'm fairly ambivalent about direct_messenger as I haven't been using it.  If it's useful to folks (and potentially is useful to me) I'd be in favor of keeping it if it's not a lot of overhead to keep it around.